requirements.txt

```diff
@@ -0,0 +1,3 @@
+cs3apis
```

Comment:
Please add these in `install_requires`

Reply:
We don't need this file. Installing with pip will already pick from `install_requires`.